buildSrc/subprojects/integration-testing/src/main/kotlin/org/gradle/gradlebuild/test/fixtures/TestFixturesPlugin.kt

```diff
@@ -60,46 +55,72 @@ open class TestFixturesPlugin : Plugin<Project> {
         configureAsConsumer()
     }
 
+
+    /**
+     * This mimics what the java-library plugin does, but creating a library of test fixtures instead.
+     */
     private
     fun Project.configureAsProducer() {
+        val testFixtures by java.sourceSets.creating
 
-        configurations {
-            create("outputDirs")
+        java.sourceSets.named<SourceSet>("test") {
+            compileClasspath += testFixtures.output
```

Comment:
Would there be any advantages in adding the `jar` as a dependency instead of the output of the source set?

Reply:
No, but I'm gonna rework this anyway :)